steering_vectors/aggregators.py

```diff
@@ -1,10 +1,33 @@
-from typing import Callable
+from typing import Callable, Union
```

Comment:
Nit: no need for `Union`, just use `|` since we only support Python 3.10+

Reply:
Resolved by [7ba358f](https://github.com/steering-vectors/steering-vectors/pull/22/commits/7ba358f8229cbe662932a7d35cffe92cefbf6e28)